### PR TITLE
Feature/Typography

### PR DESCRIPTION
**Pull request by SalonSikdar**

### CODE REVIEW

File: pages/components/Text/text.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: var(--primary-color);
+}
+
+.text-muted {
+  color: var(--light-secondary-color) !important;
```
Comment (anasm11) [inline pages/components/Text/text.css:11]: better to not use important

**Review by anasm11 — APPROVED** [review]
Usage page is missing text link. Rest all looks good.

File: css/main.css [hunk 2]
```diff
@@ -109,3 +109,11 @@ button:hover {
   -ms-flex-align: center !important;
   align-items: center !important;
```
Comment (srishtigautam21) [inline css/main.css:110]: You should not use !important as vikram told. I don't exactly remember why but it should not be used.

**Review by srishtigautam21 — APPROVED** [review]
approving it for now you can do the suggested changes in a followup PR

File: css/main.css (hunk 2 above)
Comment (SalonSikdar) [reply css/main.css:110]: Ack. I'll try to minimize the usage of !important. in following PRs

File: pages/components/Text/text.css (hunk 1 above)
Comment (SalonSikdar) [reply pages/components/Text/text.css:11]: Ack. I'll try to minimize the usage of !important. in following PRs

**Pull request merged**